gsa/src/web/pages/audits/__tests__/actions.js

```diff
@@ -229,6 +229,91 @@ describe('Audit Actions tests', () => {
     );
   });
 
+  test('should not call click handlers for stopped audit without permissions', () => {
+    const audit = Audit.fromElement({
+      status: AUDIT_STATUS.stopped,
+      alterable: '0',
+      last_report: {report: {_id: 'id'}},
+      permissions: {permission: [{name: 'get_task'}]},
+      target: {_id: 'id', name: 'target'},
+      usage_type: 'audit',
+    });
+
+    const handleAuditClone = jest.fn();
+    const handleAuditDelete = jest.fn();
+    const handleAuditDownload = jest.fn();
+    const handleAuditEdit = jest.fn();
+    const handleAuditResume = jest.fn();
+    const handleAuditStart = jest.fn();
+    const handleAuditStop = jest.fn();
+    const handleReportDownload = jest.fn();
+
+    const {render} = rendererWith({capabilities: wrongCaps});
+    const {getAllByTestId} = render(
+      <Actions
+        entity={audit}
+        gcrFormatDefined={true}
+        links={true}
+        onAuditCloneClick={handleAuditClone}
+        onAuditDeleteClick={handleAuditDelete}
+        onAuditDownloadClick={handleAuditDownload}
+        onAuditEditClick={handleAuditEdit}
+        onAuditResumeClick={handleAuditResume}
+        onAuditStartClick={handleAuditStart}
+        onAuditStopClick={handleAuditStop}
+        onReportDownloadClick={handleReportDownload}
+      />,
+    );
+
+    const icons = getAllByTestId('svg-icon');
+
+    fireEvent.click(icons[0]);
+    expect(handleAuditStart).not.toHaveBeenCalledWith(audit);
```

Comment:
Maybe test this for ´´´.not.toHaveBeenCalled()´´´, because it should not have been called with anything instead of just with this specific audit. This would also affect the following tests below.